cmd/createorupdate/createorupdate.go

```diff
@@ -349,6 +349,15 @@ func main() {
 	}
 	rpc.Authorizer = authorizer
 
+	defer func() {
```

Comment:
For completeness, this can be moved below the `strings.ToUpper(*method) == http.MethodDelete` check so that the defer statement will never run when -request=DELETE.

Reply:
Also, this can be turned to:
```go
	if *cleanup {
		defer func() {
			if err := delete(ctx, log, rpc); err != nil {
				log.Fatal(err)
			}
		}()
	}
```